config.json

```diff
@@ -41,9 +41,9 @@
     "Firefox ESR"
   ],
   "settings": {
-    "homepage_new_products_count": 12,
-    "homepage_featured_products_count": 8,
-    "homepage_top_products_count": 8,
+    "homepage_new_products_count": 5,
+    "homepage_featured_products_count": 4,
```

Comment:
FeaturedAndPopularProductsUITest failure here - number of expected needs to be updated in the test

Reply:
Yes, we'll need to update the tests when this is published.